### ISSUE TITLE
carla speak to tyrone about rodin

### ISSUE THREAD

**Issue by CIEspost** [zacck/cieproductions#53]
@zacck Albert said that we can partner with them and make demos for possible clients
We said i can talk to tyrone about this

I am a co-founder/COO of Rodin.io (Techstars Boulder'17), a development platform for creating VR apps without using Unity or any other 3D game development platform and by using just using JavaScript language. Maybe the best way to describe it is to check examples here - https://app.rodin.io

I have found information about you from 360 freelance database, and I think the content that you have created is awesome. I think most of the current 360/VR content makers do not realize what power they have. I believe that the content you have is the first step into the future of VR/AR. And I think you have awesome opportunity to move forward with this in South Africa.

We believe that in very near future many companies on the web will go into webVR or webAR. And now it is the best time to start moving clients that already have 360/VR content to right direction.

So, Carla, I don't want to sound very selling, but we would like to partner with you guys, to offer clients the webVR websites/experiences and apps now. We have big experience and capabilities of creating projects in VR and very soon in AR.

Let me know if we can jump on the call and see what we can do together in South Africa.

**Comment by zacck-zz** [comment]
@CIEspost  do you still need my involvement in this?

**Comment by CIEspost** [comment]
@zacck yes - after listening to that podcast i sent you a link of, I think we need to do a bussiness plan asap so that we know also what partnerships it is that we want and we dont just take an oppurtunity for the sake of an opportunity or free ride if you know what i mean. Please tell me what you think about that product and if the thing that we said we can gain is wirhtwhile and also...is it somehting that is alligned witht he bussiness' growth - will it steeer us futhert in the direction we want to go? Or is it going to take us on a raod that we dont necesraily wnat to be on?

**Comment by zacck-zz** [comment]
We can only answer most of these questions when doing the business plan for CIE
